src/Domain/Codec/BytesBinaryCodec.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Erdcsharp.Domain.Helper;
+using Erdcsharp.Domain.Values;
+
+namespace Erdcsharp.Domain.Codec
+{
+    public class BytesBinaryCodec : IBinaryCodec
+    {
+        public string Type => TypeValue.BinaryTypes.Bytes;
+
+        private const int BytesSizeOfU32 = 4;
```

Comment:
In C# you can use the `sizeof` operator. We didn't have it in TypeScript:

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/sizeof